lang_GENERIC/analyze/Normalize_AST.ml

```diff
@@ -32,13 +32,14 @@ open AST_generic
 (* Entry point *)
 (*****************************************************************************)
 
-let normalize2 any =
+let normalize2 any lang =
   let visitor = Map_AST.mk_visitor { Map_AST.default_visitor with Map_AST.
     kexpr = (fun (k, _) e ->
       (* apply on children *)
       let e = k e in
       match e with
-      | Call (IdSpecial (ArithOp op, tok), [a; b]) ->
+      | Call (IdSpecial (ArithOp op, tok), [a; b]) when lang <> Lang.Python ->
+        (* != can be a method call in Python *)
         let rewrite_opt =
           match op with
           | NotEq -> Some (Not, Eq)
```

Comment:
This is the line where `!=` operation is normalized. I feel the `when lang <> Lang.Python` should be on this line